docs/debugging_crashes.md

```diff
@@ -0,0 +1,53 @@
+# Debugging Crashes
+
+This document covers the following topics:
+1. [How to get a crash dump](debugging_crashes.md#How-to-get-a-crash-dump)
+2. [How to get a good callstack for crashes](debugging_crashes.md#How-to-get-a-good-callstack-for-crashes)
+
+## How to get a crash dump
+
+### Using Visual Studio
```

Comment:
Should the advise given [here](https://github.com/microsoft/microsoft-ui-xaml/issues/2056#issuecomment-594178348) be worked into this section as well?